lib/forspell/loader.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative 'loaders/markdown'
+require_relative 'loaders/ruby'
+require_relative 'loaders/c'
+
+module Forspell
+  module Loader
```

Comment:
I meant previosly, we have all loaders in `Loaders` module, so it would be just logical to have here `Loaders.for(...)` ;)